src/integration/__tests__/__fixtures__/knownV2Failures.json

```diff
@@ -955,5 +955,7 @@
   "491575e8d9b9f682de1cb678fd351c4b": null,
   "e089beca2667c383e7e2c99c42e8750f": null,
   "bd37ef982226143b0d68d972b5fee276": null,
-  "5d6a47112815cab926fb8f2036b94039": null
+  "5d6a47112815cab926fb8f2036b94039": null,
+  "cfac0c7e54c2601783e10bca99bb8db7": null,
+  "956ec3ea65b6fb1a68dabdf45a966f9f": null
```

Comment:
TIL we have this pattern for marking known failures in the case that a persisted query is no longer supported due to breaking schema changes.

Discovered due to [the earlier build](https://app.circleci.com/pipelines/github/artsy/metaphysics?branch=roop%2FFX-3041-disable-searches) fails on this PR